readme.md

```diff
@@ -469,6 +469,39 @@ Inject a function to be executed prior to navigation.
 
 This can be useful for [altering the JavaScript environment](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageevaluateonnewdocumentpagefunction-args). For example, you could define a global method on the `window`, overwrite `navigator.languages` to change the language presented by the browser, or mock `Math.random` to return a fixed value.
 
+##### clip
+
+Type: `object`
+
```

Comment:
It should be documented whether the Y position starts from the top or bottom of the website.